tasks/test-changes.js

```diff
@@ -23,7 +23,7 @@ const getChangedPackages = () => {
     let command;
     try {
         command = execSync(
-            'yarn --silent lerna ls --since origin/main --json --loglevel silent'
+            'yarn --silent lerna ls --since origin/main --json --loglevel silent --ignore "@swc-react/*"'
```

Comment:
I added this here for the short term. BUT, if we choose to make these packages build artifacts that aren't tracked, we might not need this 🤔